hack/test-extended/default/run.sh

```diff
@@ -108,6 +89,13 @@ done <<< "${ALL_IP_ADDRESSES}"
 
 configure_os_server
 
+echo "[INFO] `openshift version`"
```

Comment:
We have standard names for these args across all scripts now, right?  That would allow us to move this to `start_os_server`